web/cypress/integration/morecast-page.spec.ts

```diff
@@ -1,6 +1,24 @@
 import { FIRE_WEATHER_ROUTE, MORECAST_ROUTE, PARTIAL_WIDTH } from '../../src/utils/constants'
 import { stationCodeQueryKey, timeOfInterestQueryKey } from '../../src/utils/url'
 const stationCode = 328
+const stationCode2 = 380
+const numOfObservations = 119
+
+const interceptData = () => {
+  cy.intercept('POST', 'api/observations/', { fixture: 'weather-data/observations' })
+  cy.intercept('POST', 'api/forecasts/noon/', { fixture: 'weather-data/noon-forecasts' })
+  cy.intercept('POST', 'api/forecasts/noon/summaries/', { fixture: 'weather-data/noon-forecast-summaries' })
+  cy.intercept('POST', 'api/weather_models/GDPS/predictions/most_recent', {fixture:'weather-data/models-with-bias-adjusted'}) // prettier-ignore
```

Comment:
are these fixture names for model API calls correct? Are they all actually bias-adjusted?

Reply:
I don't really care that much about what's in them - tests aren't interrogating to that level. So I haven't needed to peek inside the data all.